### PR TITLE
[NRH-342] Footeresque Yearly text

### PR DESCRIPTION
**Pull request by michael-caktus**
#### What's this PR do?
Updates footer-esque static text with new text for yearly contributions, as per [this basecamp post](https://3.basecamp.com/3120230/buckets/21497549/messages/4101300434).

#### How should this be manually tested?
Go to a live donation page and toggle between one-time, monthly, and yearly donations observing the text changes at the bottom of the page.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No

### CODE REVIEW

File: spa/cypress/integration/donation-page.spec.js [hunk 1]
```diff
@@ -385,7 +388,7 @@ describe('Donation page', () => {
     });
   });
 
-  describe('Footer-like content', () => {
+  describe.only('Footer-like content', () => {
```
Comment (JeremyGibson) [inline spa/cypress/integration/donation-page.spec.js:391]: Only needs removing

File: spa/src/components/donationPage/DonationPageStaticText.js [hunk 2]
```diff
@@ -4,6 +4,21 @@ import { format } from 'date-fns';
 import { getTotalAmount } from 'components/paymentProviders/stripe/stripeFns';
 
 function DonationPageStaticText({ page, amount, payFee, frequency }) {
+  const getFreqDependentText = () => {
+    console.log('hey that freq: ', frequency);
```
Comment (JeremyGibson) [inline spa/src/components/donationPage/DonationPageStaticText.js:8]: Not sure if you want to leave that in here.

File: spa/src/components/donationPage/DonationPageStaticText.js [hunk 3]
```diff
@@ -4,6 +4,21 @@ import { format } from 'date-fns';
...
+    if (frequency === 'one_time') return null;
+    let freqDependentText = ' ';
+    const totalAmount = getTotalAmount(amount, payFee, frequency, page.organization_is_nonprofit);
+    freqDependentText += `Additionally, by proceeding with this transaction, you're authorizing today's payment, along with all future recurring payments of $${totalAmount}, to be processed on or adjacent to `;
```
Comment (JeremyGibson) [inline spa/src/components/donationPage/DonationPageStaticText.js:12]: Should this  be two $$?

**Review by JeremyGibson — CHANGES_REQUESTED** [review]
Works fine just some small cleanup.

File: spa/src/components/donationPage/DonationPageStaticText.js (hunk 3 above)
Comment (michael-caktus) [reply spa/src/components/donationPage/DonationPageStaticText.js:11]: Yeah, the first one is a literal "$". The second is for js string template literal syntax.

**Comment by michael-caktus** [comment]
@JeremyGibson Should be ready for another peek

**Review by JeremyGibson — APPROVED** [review]

**Comment by JeremyGibson** [comment]
Approved


*Jeremy Gibson*Developer

***@***.*** ***@***.***>
www.caktusgroup.com
<http://www.google.com/url?q=http%3A%2F%2Fwww.caktusgroup.com%2F&sa=D&sntz=1&usg=AFQjCNG6RpHdHUA5QLae7099-vwv2NLrvg>

**Pull request merged**